### PR TITLE
Remove legacy compliance form from exercise flow.

### PR DESCRIPTION
**Pull request by eschlange**
* Added test gems for feature specs.
* Added an authentication helper module for feature specs.
* Pulled external JS reference into vendor asset path.
* Updated issue with default images in several models.
* Removed legacy compliance routes.
* Removed legacy compliance controller.
* Updated rails helper with feature spec config.
* Added seed fixtures for testing features.
* Updated existing feature specs with valid data.
* Fixed Capybara issue with TimeCop warning in exercise log spec.
* Updated all necessary views to bypass the legacy compliance form.
* Updated devise config to allow for using GET to sign out in testing.

[#113878391]

### CODE REVIEW

File: app/models/flash_card.rb [hunk 1]
```diff
@@ -3,7 +3,9 @@ class FlashCard < ActiveRecord::Base
 	belongs_to :user
 	has_many :responses
 
-	has_attached_file :card_image, :styles => { :medium => "300x300>", :thumb => "100x100>" }, :default_url => "/images/default_image.png"
-	
-	
+	has_attached_file :card_image,
```
Comment (michaelwehrley) [inline app/models/flash_card.rb]: I think RubyMine goofed up the styling again.  Can you fix this?  Thanks!

File: app/assets/javascripts/picture_deck.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ function concludeExercise(hintCount, timeStart, response) {
       $(this).attr('id') === 'restart' ? options.restart = true : options.last_card = true;
       postResponse(responseObject, options);
     } else {
-      location.href='/user/'+app.current_user_id+'/assignments/'+app.assignment_id+'/compliance_forms/new';
+      location.href='/';
```
Comment (michaelwehrley) [inline app/assets/javascripts/picture_deck.js]: @eschlange any need to test these new href changes?

**Comment by michaelwehrley** [comment]
:+1:

File: app/assets/javascripts/picture_deck.js (hunk 2 above)
Comment (eschlange) [inline app/assets/javascripts/picture_deck.js]: The feature spec added in this PR tests this change.

**Pull request merged**